server/extensions/component-jsx.js

```diff
@@ -16,8 +16,13 @@ export default class Component {
     return new nodes.CallExtension(this, 'run', args);
   };
 
-  run(_/* context */, name, model = {}) {
-    const html = ReactDOMServer.renderToString(components[name](model));
+  run(_/* context */, name, model = {}, children = []) {
+    const childrenComponents = children
+      .filter(_ => _)
```

Comment:
Why are we passing objects that are `null` through?

Reply:
I think I see why... is it that we might not have the TASL component?

Reply:
yep, you got it.